sdk/storage/Azure.Storage.Common/src/Sas/AccountSasPermissions.cs

```diff
@@ -56,6 +56,24 @@ public enum AccountSasPermissions
         /// </summary>
         Process = 128,
 
+        /// <summary>
+        /// Indicates that reading and writing Tags is permitted.
+        /// Blob service only.
+        /// </summary>
+        Tag = 256,
```

Comment:
I went with more generic names in `AccountSasPermissions`, in case another service ever gets similar features with SAS permissions.